pkg/actor/actor.go

```diff
@@ -18,6 +18,7 @@ package actor
 
 import (
 	"context"
```

Comment:
Can you remove this please?

Reply:
You mean the new line? The new `make dev/fmt` command added that.